package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "discord_bots",
-  "version": "0.4.1",
+  "version": "0.5.0",
```

Comment:
```suggestion
  "version": "0.4.9",
```

Reply:
This PR adds a new feature. Changing the version to 0.5.0 is correct.

Reply:
![image](https://user-images.githubusercontent.com/24265894/57270947-7d152100-70e1-11e9-885d-77cc32aea338.png)
So this was you talking without thinking, then?

Reply:
v0.5.0 is only released when we add the tag to the repo. The version number on the master branch should always be the correct number if we were to release right now. (To prevent having to make "version bump" commits.)

Since this PR adds new features, the bot would be on v0.5.0 if we released a new version immediately after merging. So the version in the package.json should be changed to 0.5.0 in this PR.

Reply:
Even so, if you were to add a new feature that would make it v0.6.0, no?

Reply:
Not if v0.5.0 hasn't been released yet. v0.5.0 can have any number of additional features over v0.4.x, not just one.

If the last release is v0.4.1 and I'm adding a new feature, the PR will set the version in package.json to v0.5.0. The package.json in the master branch is not relevant. (The only exception is if the version number in the package.json file is already higher. I can't lower it.)

Similarly, if the last release is v0.4.1 and I'm doing a patch (that doesn't add features), the PR will set the version in the package.json to v0.4.2 (unless it is already higher). In case of a merge conflict, the higher number prevails.

Reply:
Yes, but if Tanner's commit makes it v0.5.0 and it gets pulled to master, then your commit can't make it v0.5.0 because that's already the released version, right?

Reply:
No, the released version only becomes 0.5.0 once we push a "v0.5.0" tag to the repo.